src/bic/collection/form-records.js

```diff
@@ -33,14 +33,14 @@ define(function (require) {
 
             collection.reset();
 
-            nodes = data.evaluate('//' + formName, data);
+            nodes = data.evaluate('//' + formName, data, null, XPathResult.ANY_TYPE, null);
             node = nodes.iterateNext();
 
             parseNodes = function (key) {
               if (key.nodeName === 'id') {
-                parsed.id = key.innerHTML;
+                parsed.id = key.textContent;
```

Comment:
So customers can't customise their XML output? It's always free of non-data spaces?

I've just been playing with some of the XML fixtures in the Forms library, and those all happen to be human-friendly. Maybe "human-friendly" is not what I mean, perhaps "pretty-printed"?